### DISCUSSION TITLE
On the Customisations page, trans-rights-scarf displays in completely the wrong colours when in dark mode

### DISCUSSION THREAD

**Discussion by Starwort** [BattlesnakeOfficial/feedback#243] [Bug Reports]
![image](https://github.com/BattlesnakeOfficial/feedback/assets/16487249/3399f4e5-f7b3-4cd9-bf43-95248e9f1228)

**Comment by bvanvugt** [comment]
Thanks for reporting, will take a look! :+1:

**Comment by bvanvugt** [comment]
@Starwort fixed! Not as elegant as I'd like, but at least not broken.
<img width="547" alt="Screenshot 2023-09-09 at 7 56 55 PM" src="https://github.com/BattlesnakeOfficial/feedback/assets/1531419/f26047f1-ab90-4bab-b48b-95b663846258">
